django_statsd/clients/null.py

```diff
@@ -1,8 +1,12 @@
+from __future__ import absolute_import
 from statsd.client import StatsClient
 
 
 class StatsClient(StatsClient):
     """A null client that does nothing."""
 
+    def __init__(self, host, port, prefix=None, **kwargs):
+        self._prefix = prefix
```

Comment:
What's the purpose of `_prefix` ?

Reply:
StatsClient expects that the `_prefix` exists. The purpose of adding the __init__ is so it doesn't try to setup a socket connection using this client.